frontend/src/api/auth/request.ts

```diff
@@ -0,0 +1,29 @@
+import client from '../client';
+
+interface storeLoginType {
+  storeId: string;
+  password: string;
+}
+
+interface storeRegisterType {
+  storeId: string;
+  password: string;
+  branchName: string;
+}
+
+export async function login(store: storeLoginType) {
```

Comment:
```suggestion
export async function login(store: storeLoginType): Promise<string> {
```
이런 식으로 data의 type을 지정하는건 어떠신가요?

Reply:
코드 표현력이 훨씬 높아보이네요!